examples/rosettacode/100_doors.nit

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: 100 doors
+# SEE: <http://rosettacode.org/wiki/100_doors>
+
+# A door with two states : open or close
+class Door
+	#True : Open / False : Close
+	var state = false
+
+	fun open do state = true
```

Comment:
You could compile the last 2 lines into a single attribute:

`var open = false`

And use open instead of state everywhere.

In general, the name `state` for an attribute is not precise enough. The attributes are the state of the instance by definition, the name of the attribute should instead explain its role further.